Flow Actions/Set Location/Readme.ms

```diff
@@ -0,0 +1 @@
+This will populate a <u_location> field string based on the <location> field on the user's record without having to use GlideAjax. Go to the <u_location> dictionary and place this snippet in the Default Value field.
```

Comment:
Invalid file type, `u_installation_id` from the script is not documented, and this readme does not describe much at all

Reply:
Sorry, I looked at another readme from someone else and though I followed what they did. I'll add more detail. 

As for the filetype, I followed the contribution file but for a newb like me, it doesn't provide the level of detail I need. I simply added a file as it states and typed away.

As for the field. I'm going to add in the Readme this applies to a reference field that is added. "u_installation" being the name of my particular field. The <> was an attempt to say "whatever you want to name it".